rootfs/etc/cont-init.d/01-acarshub

```diff
@@ -26,12 +38,27 @@ if [ -n "${ENABLE_VDLM}" ]; then
 		echo "Error: VDLM enabled with no STATION_ID_VDLM set. Exiting"
 		exit 1
 	fi
-	if [ -z "${FREQS_VDLM}" ]; then
-		echo "Error: VDLM enabled with no FREQS_VDLM. Exiting"
+	if [ -n "${FREQS_VDLM}" ]; then
+		FREQ_CMD+=("-v" $(echo "$FREQS_VDLM" | sed 's/;/ /g'))
+	elif [ -z "${VDLM_FREQ_0}" ]; then
+		echo "Error: VDLM enabled with no FREQS_VDLM or VDLM_FREQ_0. Exiting"
 		exit 1
 	fi
 fi
 
+if [ -n "${SERIAL}" ]; then
+	FREQ_CMD+=("-s" $(echo "$SERIAL" | sed 's/;/ /g'))
+fi
+
+echo "${FREQ_CMD[@]}"
+echo "Generating decoder startup files"
+
+pushd ../../scripts || exit
```

Comment:
I think `exit` needs to be `exit 1`, so s6-init is aware that there was a problem...

Reply:
That makes sense...fixed